code/modules/mob/living/carbon/superior_animal/roach/blattedin.dm

```diff
@@ -0,0 +1,10 @@
+/datum/reagent/toxin/blattedin/touch_mob(var/mob/living/carbon/superior_animal/roach/bug, amount)
```

Comment:
Does this reviving actually work now?

Reply:
It should apply the same as acid spraying which I test so yes. Though I did not actually test the revival.